test/core/primitives/block/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+addtest(block_test
+    block_test.cpp
+    )
+target_link_libraries(block_test
+    address
```

Comment:
tests should be linked with as little libraries number as possible. ideally, with only one.
if libraries are required for the library being tested, they should be linked to it, not to the test.
I mean address, cbor and tockets libraries.

Reply:
I see, there is no block library. maybe add such library and link all the dependencies to it>